staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -163,6 +166,8 @@ func WithPriorityAndFairness(
 		}
 
 		// find the estimated "width" of the request
+		// TODO: Maybe just make it costEstimator and let it return additionalLatency too for the watch?
+		// TODO: Estimate cost should also take fcIfc.GetWatchCount(requestInfo) as a parameter.
```

Comment:
`requestInfo` can be obtained from the request context, so i don't think we will need to change signature of `EstimateWidth`

Reply:
Yes - I will address that as part of https://github.com/kubernetes/kubernetes/pull/103539